### PR TITLE
Pengu's Wintercoats V4

### PR DESCRIPTION
**Pull request by Homingpenguins**
## About The Pull Request
This adds 4 winter coats to the game.
Chemist, Geneticist, Bartender, and Lawyer.
Credit for the coat sprites go to Jo, YouTubeBoy, and BeardlessJo. Thanks for the sprites.

## Why It's Good For The Game
It add another 4 coats, bringing us closer to the ultimate goal of all jobs having a winter coat.

## Changelog
:cl:
imageadd: added some icons and images; the 4 coats.
code: changed some code; code>fulpcode>wintercoats.dm
/:cl:

### CODE REVIEW

**Comment by Homingpenguins** [comment]
@TheSwain sorry for the new pr. because of the way i made the first pr, i had to make another to edit it. problems with using forks to make prs. I fixed the position of the coats in the vending machines, which is what you asked for. this should be ready to go.

**Comment by TheSwain** [comment]
Almost there. The vending machine lists don't have a close parenthesis. if you test this i guarantee you'll get an error.

**Comment by Homingpenguins** [comment]
alright. That should fix it, but ill test it anyways. ill be back soon. or even just compile it.

**Comment by Homingpenguins** [comment]
i just tested it on a private server. it also compiled in DM. if its still messed up somehow. i must be blind.

File: code/FulpstationCode/wintercoats.dm [hunk 1]
```diff
@@ -104,3 +104,79 @@
...
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/security/law
+
+/obj/item/clothing/head/hooded/winterhood/security/law
+	mob_overlay_icon = 'icons/fulpicons/Penguins_cove/Winter_Coat_Pile/wintercoats.dmi'
+	icon = 'icons/fulpicons/Penguins_cove/Winter_Coat_Pile/wintercoathoods.dmi'
+	icon_state = "winterhood_law"
+
+/obj/item/clothing/suit/hooded/wintercoat/service/bartender
+	name = "\improper Bartender's winter coat"
+	desc = "This is a winter coat made to look like a butlar's suit."
```
Comment (SomeAngryMiner) [inline code/FulpstationCode/wintercoats.dm]: Pretty sure its "butler", not "butlar".

File: code/FulpstationCode/wintercoats.dm [hunk 2]
```diff
@@ -104,3 +104,79 @@
...
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/service/bar
+
+/obj/item/clothing/head/hooded/winterhood/service/bar
+	mob_overlay_icon = 'icons/fulpicons/Penguins_cove/Winter_Coat_Pile/wintercoats.dmi'
+	icon = 'icons/fulpicons/Penguins_cove/Winter_Coat_Pile/wintercoathoods.dmi'
+	icon_state = "winterhood_bar"
+
+/obj/item/clothing/suit/hooded/wintercoat/medical/chem
+	name = "\improper Chemist's winter coat"
+	desc = "This is a winter coat made protect from minor chemical spills and to have a styalish orange theme."
```
Comment (SomeAngryMiner) [inline code/FulpstationCode/wintercoats.dm]: have an extra "A" on stylish, mate.

File: code/FulpstationCode/wintercoats.dm [hunk 3]
```diff
@@ -104,3 +104,79 @@
...
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/medical/chem
+
+/obj/item/clothing/head/hooded/winterhood/medical/chem
+	mob_overlay_icon = 'icons/fulpicons/Penguins_cove/Winter_Coat_Pile/wintercoats.dmi'
+	icon = 'icons/fulpicons/Penguins_cove/Winter_Coat_Pile/wintercoathoods.dmi'
+	icon_state = "winterhood_chem"
+
+/obj/item/clothing/suit/hooded/wintercoat/medical/gen
+	name = "\improper Geneticist's winter coat"
+	desc = "This winter coat is made out of a comfortable material and dyed to the geneticest's color scheme."
```
Comment (SomeAngryMiner) [inline code/FulpstationCode/wintercoats.dm]: change this "geneticest" to "geneticist", and should be good.

**Comment by SomeAngryMiner** [comment]
fix those typos, and personally think it should be good to go.
Not really fond of the comments on the locker stuff, and would rename all the pathing from "penguins_cove" folder to simply "wintercoats", but thats a personal choice.

**Comment by Homingpenguins** [comment]
The pengu's cove is gor all my icons that i make. I have more going in that folder soon. And as for the typos. Im not that good at spelling. Ill fix them when i get home. @SomeAngryMiner.

**Comment by Homingpenguins** [comment]
that fix places the coats in the correct spot in the vending machine code. i opened the pins in the dev channel, and found Swain's guide to that exact thing...

**Pull request closed without merging**